lambdas/src/services/session-request-validator.ts

```diff
@@ -27,17 +27,22 @@ export class SessionRequestValidator {
         const evidenceRequested = payload["evidence_requested"] as EvidenceRequest;
         const state = payload["state"] as string;
 
-        if (evidenceRequested && evidenceRequested.scoringPolicy !== "gpg45") {
+        if (evidenceRequested && evidenceRequested?.scoringPolicy !== "gpg45") {
```

Comment:
[nit] why do we need the optional chain? We have already confirmed that `evidenceRequested` is truthy in the first half of this statement, no?